Tools/_sunrise/pr_check_summary.py

```
@@ -0,0 +1,726 @@
+def main() -> int:
+    args = parse_args()
+    token = os.environ.get("GITHUB_TOKEN")
+    repository = os.environ.get("GITHUB_REPOSITORY")
+    event_name = os.environ.get("GITHUB_EVENT_NAME", "")
+    event_path = os.environ.get("GITHUB_EVENT_PATH")
+
+    if not token or not repository or not event_path:
+        print("GITHUB_TOKEN, GITHUB_REPOSITORY and GITHUB_EVENT_PATH are required.", file=sys.stderr)
+        return 1
+
+    with open(event_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)